v3_map.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2012 The GoSNMP Authors. All rights reserved.  Use of this
+// source code is governed by a BSD-style license that can be found in the
+// LICENSE file.
+
+// Copyright 2009 The Go Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
```

Comment:
Why do the go authors also have copyright?

Reply:
I think I just copy pasted from the `v3.go` file 😅 I'll update this, wasn't thinking too deeply about it when I made this file! I will amend to remove this aspect.